### ISSUE TITLE
Does not parse arXiv emails with "empty" footer

### ISSUE THREAD

**Issue by louiskirsch** [deragent/arXivFilter#7]
When pasting the arxiv email, I get 'This is not an arxiv email!'
```
------------------------------------------------------------------------------
------------------------------------------------------------------------------
Send any comments regarding submissions directly to submitter.
------------------------------------------------------------------------------
Archives at http://arxiv.org/
To unsubscribe, e-mail To: [cs@arXiv.org](mailto:cs@arXiv.org), Subject: cancel
------------------------------------------------------------------------------
Submissions to:
Artificial Intelligence
Machine Learning
received from  Mon 28 Aug 23 18:00:00 GMT  to  Tue 29 Aug 23 18:00:00 GMT
------------------------------------------------------------------------------
------------------------------------------------------------------------------
\\
arXiv:2308.14815
Date: Mon, 28 Aug 2023 18:06:24 GMT   (3725kb,D)

Title: Distributionally Robust Statistical Verification with Imprecise Neural
 Networks
Authors: Souradeep Dutta, Michele Caprio, Vivian Lin, Matthew Cleaveland, Kuk
 Jin Jang, Ivan Ruchkin, Oleg Sokolsky, Insup Lee
Categories: cs.AI cs.LG cs.RO
\\
 A particularly challenging problem in AI safety is providing guarantees on
the behavior of high-dimensional autonomous systems. Verification approaches
centered around reachability analysis fail to scale, and purely statistical
approaches are constrained by the distributional assumptions about the sampling
process. Instead, we pose a distributionally robust version of the statistical
verification problem for black-box systems, where our performance guarantees
hold over a large family of distributions. This paper proposes a novel approach
based on a combination of active learning, uncertainty quantification, and
neural network verification. A central piece of our approach is an ensemble
technique called Imprecise Neural Networks, which provides the uncertainty to
guide active learning. The active learning uses an exhaustive neural-network
verification tool Sherlock to collect samples. An evaluation on multiple
physical simulators in the openAI gym Mujoco environments with
reinforcement-learned controllers demonstrates that our approach can provide
useful and scalable guarantees for high-dimensional systems.
\\ ( https://arxiv.org/abs/2308.14815 ,  3725kb)
------------------------------------------------------------------------------
```

**Comment by deragent** [comment]
Is this the full arxiv-email that you receive?

In my case, there is usually also a footer at the bottom, in the form:

    %%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---
    For subscribe options to combined physics archives,
    e-mail To: [physics@arxiv.org](mailto:physics@arxiv.org), Subject: subscribe
    -----------------------------------------------------------------------------
    For help on viewing and making submissions, see http://arxiv.org/help/

And the parser also looks for, and removes this part during the parsing.

**Comment by louiskirsch** [comment]
This is the footer, it looks a bit different from yours, maybe that is the issue.
```
------------------------------------------------------------------------------
\\
arXiv:2308.12896
replaced with revised version Tue, 29 Aug 2023 15:57:02 GMT   (2999kb,D)

Title: Beyond Document Page Classification: Design, Datasets, and Challenges
Authors: Jordy Van Landeghem, Sanket Biswas, Matthew B. Blaschko,
 Marie-Francine Moens
Categories: cs.CV cs.CL cs.LG
Comments: 8 pages, under review
\\ ( https://arxiv.org/abs/2308.12896 ,  2999kb)
%%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---%%%---
```

**Comment by deragent** [comment]
I'll look into this, but this should be an easy fix!

Could you maybe attach a full copy of one of your E-Mails, for testing?

**Comment by louiskirsch** [comment]
Thanks!
[arxiv.txt](https://github.com/deragent/arXivFilter/files/12474181/arxiv.txt)

**Comment by deragent** [comment]
So, the E-Mail with the simple footer should no longer lead to an error.

See version 0.3.4: https://pypi.org/project/arxiv-filter/0.3.4/

@louiskirsch: On other thing I noticed in the `arxiv.txt` file that you posted, is that there are a lot of newlines in unexpected places. For example the author lists are often on multiple lines (see lines 46 - 50).
Is this always the case in your E-mails, or is this due to the way you copied the E-Mail?

The current version of arxiv_filter can not handle such newlines, and thus will not properly parse the author lists in the `arxiv.txt` file which you posted.

**Comment by louiskirsch** [comment]
I checked the source of the email, these line breaks (in the authors list) are in the original email I received.

**Comment by deragent** [comment]
Indeed, I just checked that this is similar in my emails, but there seems to be a difference in the number of spaces at the beginning of a line which continues the previous line. I created a new issue for this: #8 

But the original issue should be fixed with the latest release (0.3.4) - closing.
